content/en/profiler/enabling/ddprof.md

```diff
@@ -217,26 +195,15 @@ You can configure logging to one of several endpoints:
 - `disable` disables the logs entirely.
 - Any other value is treated as a file path, with a leading `/` designating an absolute path.
 
-### Pidmode
```

Comment:
issue on pidmode should be resolved before we advertise this again 
https://github.com/DataDog/ddprof/issues/283